src/main/java/com/sweng894/GetVaccinated/api/repository/AppointmentRepository.java

```diff
@@ -1,17 +1,27 @@
 package com.sweng894.GetVaccinated.api.repository;
 
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
 import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBSaveExpression;
 import com.amazonaws.services.dynamodbv2.model.AttributeValue;
 import com.amazonaws.services.dynamodbv2.model.ExpectedAttributeValue;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Repository;
+import com.amazonaws.services.dynamodbv2.model.ResourceInUseException;
 import com.sweng894.GetVaccinated.api.entity.Appointment;
+import org.springframework.stereotype.Repository;
 
 @Repository
 public class AppointmentRepository {
-  @Autowired
-  private DynamoDBMapper dynamoDBMapper;
+  private final DynamoDBMapper dynamoDBMapper;
+
+  public AppointmentRepository(DynamoDBMapper dynamoDBMapper, AmazonDynamoDB dynamoDB) {
```

Comment:
Moved to constructor-based instead of field-based dependency injection. Also, not sure if it's the best place for it, but creation of the tables required for the repository are being handled as part of its constructor.